pkg/chartverifier/certifier.go

```diff
@@ -36,17 +36,26 @@ func (e CheckErr) Error() string {
 	return "check error: " + string(e)
 }
 
+type OpenShiftVersionErr struct {
+	msg string
+}
```

Comment:
This could be changed into `type OpenShiftVersionErr string` instead, since `msg` is the only field in the type.

Reply:
Done. bd97de524900698c5badb59f30df389f1b827d63